src/Controller/Rest/AbstractRestController.php

```diff
@@ -50,7 +50,7 @@ public function needsSessionDoubleAuthenticationCheck(): bool
     /**
      * @inheritDoc
      */
-    protected function checkPermission($permission): void
+    protected function checkPermission(string $permission): void
```

Comment:
```suggestion
    protected function checkPermission($permission): void
```
Breaks support for 10.6